core/src/elements/kick_bomb.rs

```diff
@@ -177,21 +197,19 @@ fn update_lit_kick_bombs(
             explosion_sound,
             explosion_volume,
             kick_velocity,
-            fuse_time,
             damage_region_lifetime,
             damage_region_size,
             explosion_lifetime,
             explosion_atlas,
             explosion_fps,
             explosion_frames,
-            arm_delay,
             fin_anim,
             ..
         } = &element_meta.builtin else {
             unreachable!();
         };
 
-        kick_bomb.age += 1.0 / crate::FPS;
+        kick_bomb.fuse_time.tick(time.delta());
```

Comment:
We need to make sure we tick both timers here:

```suggestion
        kick_bomb.fuse_time.tick(time.delta());
        kick_bomb.arm_delay.tick(time.delta());        
```